tests/testthat/test-broom-als.R

```diff
@@ -0,0 +1,68 @@
+context("broom-als")
+
+test_that("als.tidy() works", {
+  sc <- testthat_spark_connection()
+  test_requires_version("2.0.0")
+
+  movies <- data.frame(user   = c(1, 2, 0, 1, 2, 0),
+                       item   = c(1, 1, 1, 2, 2, 0),
+                       rating = c(3, 1, 2, 4, 5, 4))
+
+  movies_tbl <- sdf_copy_to(sc,
+                            movies,
+                            name = "movies_tbl",
+                            overwrite = TRUE)
+
+  td1 <- movies_tbl %>%
+    ml_als(rating ~ user + item) %>%
+    tidy() %>%
+    dplyr::collect()
+
+  check_tidy(td1, exp.row = 3, exp.col = 3,
+             exp.names = c("id", "user_factors", "item_factors"))
+
+  expect_equal(td1$id, c(2, 0 , 1))
+})
+
+test_that("als.augment() works", {
+  test_requires_version("2.0.0")
+  sc <- testthat_spark_connection()
+
+  movies <- data.frame(user   = c(1, 2, 0, 1, 2, 0),
+                       item   = c(1, 1, 1, 2, 2, 0),
+                       rating = c(3, 1, 2, 4, 5, 4))
+
+  movies_tbl <- sdf_copy_to(sc,
```

Comment:
Let's use `copy_to()` (which is now re-exported) throughout since we'll be deprecating `sdf_copy_to()`.